1.9/api/master-routes.md

```diff
@@ -0,0 +1,18 @@
+---
+post_title: Master Routes
+menu_order: 1
+---
+
+Admin Router runs on DC/OS master nodes and exposes the following API routes.
```

Comment:
"Admin Router" by now is a name just like "Cosmos". Let's not use an article in this case. That should as of now match significant parts of the remaining documentation.